crates/re_renderer/examples/depth_cloud.rs

```diff
@@ -0,0 +1,458 @@
+        resolution_in_pixel: [u32; 2],
+        target_location: glam::Vec2,
+        frame_draw_data: &FD,
+        image_draw_data: &ID,
+    ) -> framework::ViewDrawResult
+    where
+        FD: DrawData + Sync + Send + Clone + 'static,
+        ID: DrawData + Sync + Send + Clone + 'static,
+    {
+        let Self {
+            depth,
+            scale,
+            radius_scale,
+            intrinsics,
+            ..
+        } = self;
+
+        let world_from_obj = glam::Mat4::from_cols(
+            glam::Vec4::NEG_X * *scale,
+            glam::Vec4::NEG_Y * *scale,
+            glam::Vec4::Z * *scale,
+            glam::Vec4::W,
+        );
+
+        let depth_cloud_draw_data = DepthCloudDrawData::new(
+            re_ctx,
+            &[DepthCloud {
+                world_from_obj,
+                intrinsics: *intrinsics,
+                radius_scale: *radius_scale,
+                depth_dimensions: depth.dimensions,
+                depth_data: depth.data.clone(),
+            }],
+        )
+        .unwrap();
+
+        let mut view_builder = ViewBuilder::default();
+        view_builder
+            .setup_view(
+                re_ctx,
+                view_builder::TargetConfiguration {
+                    name: "Depth Cloud".into(),
+                    resolution_in_pixel,
+                    view_from_world: IsoTransform::look_at_rh(
+                        self.camera_position,
+                        Vec3::ZERO,
+                        Vec3::Y,
+                    )
+                    .unwrap(),
+                    projection_from_view: Projection::Perspective {
+                        vertical_fov: 70.0 * std::f32::consts::TAU / 360.0,
+                        near_plane_distance: 0.01,
+                    },
+                    pixels_from_point,
+                    ..Default::default()
+                },
+            )
+            .unwrap();
+
+        let command_buffer = view_builder
+            .queue_draw(&GenericSkyboxDrawData::new(re_ctx))
+            .queue_draw(&depth_cloud_draw_data)
+            .queue_draw(frame_draw_data)
+            .queue_draw(image_draw_data)
+            .draw(re_ctx, ecolor::Rgba::TRANSPARENT)
+            .unwrap();
+
+        framework::ViewDrawResult {
+            view_builder,
+            command_buffer,
+            target_location,
+        }
+    }
+}
+
+impl framework::Example for RenderDepthClouds {
+    fn title() -> &'static str {
+        "Depth clouds"
+    }
+
+    fn new(re_ctx: &mut re_renderer::RenderContext) -> Self {
+        re_log::info!("Stop camera movement by pressing 'Space'");
+
+        let depth = DepthTexture::from_bytes(include_bytes!("assets/nyud_depth.png"), None);
+        let albedo = AlbedoTexture::from_bytes(
+            include_bytes!("assets/nyud_albedo.png"),
+            depth.dimensions.into(),
+        );
+
+        let albedo_handle = re_ctx.texture_manager_2d.create(
+            &mut re_ctx.gpu_resources.textures,
```

Comment:
maybe we can share this code with the 2d.rs sample where there's also a texture being loaded in like that

Reply:
I'll add a procedural one it's no biggie